costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -126,8 +127,14 @@ void InflationLayer::matchSize()
 void InflationLayer::updateBounds(double robot_x, double robot_y, double robot_yaw, double* min_x,
                                            double* min_y, double* max_x, double* max_y)
 {
+  static double last_min_x =  -std::numeric_limits<float>::max();
```

Comment:
won't this variable be shared across all instances of the InflationLayer -- for instance, both the local and global. Maybe pluginlib saves us from the global/local planner sharing the same values, but this should be a class member variable rather than a static, right?